src/common/PresenceStatusIndicator.js

```diff
@@ -110,6 +128,22 @@ class PresenceStatusIndicator extends PureComponent<Props> {
       return null;
     }
 
+    if (isUserAvatar && status === 'active') {
```

Comment:
Hmm, what's the reason that `isUserAvatar` is only checked for the `active` and `idle` statuses? How about `offline` and `unavailable` too?

Reply:
for offline and unavailable statuses there is no need of white background required

Reply:
Let's do a white background for all four statuses. Like you've done in your "after" screenshots from https://github.com/zulip/zulip-mobile/pull/4324#issue-530742703, right? (Or, if not: can you say why you don't want to do it for "offline" or "unavailable" anymore? 🙂) Quoting from your message:

> ### Before
> ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_32_35 (2)](https://user-images.githubusercontent.com/56453541/100827358-8a8fba00-3482-11eb-9be5-70e972667e45.png) ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_34_23 (2)](https://user-images.githubusercontent.com/56453541/100827364-8bc0e700-3482-11eb-8e37-e5540442a207.png) ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_33_59 (2)](https://user-images.githubusercontent.com/56453541/100827366-8bc0e700-3482-11eb-9eb0-9a699cbbbdbb.png) ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_32_55 (2)](https://user-images.githubusercontent.com/56453541/100827368-8c597d80-3482-11eb-9ee9-94de0eabafc5.png)
> 
> ### After
> ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_22_58 (2)](https://user-images.githubusercontent.com/56453541/100827431-b1e68700-3482-11eb-9a2e-e7fa8a27f3f5.png) ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_23_27 (2)](https://user-images.githubusercontent.com/56453541/100827432-b317b400-3482-11eb-9f56-17b212b33b1d.png) ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_25_15 (2)](https://user-images.githubusercontent.com/56453541/100827436-b3b04a80-3482-11eb-8393-1e2eecec542c.png) ![Android Emulator - Pixel_3_API_29_5554 02-12-2020 09_26_06 (2)](https://user-images.githubusercontent.com/56453541/100827437-b3b04a80-3482-11eb-98f1-3cc82ae68c44.png)

Under "After", I see an opaque white background for all status indicators.

-----

I noticed that, in this revision, the "unavailable" indicator still doesn't stand out very well:

![image](https://user-images.githubusercontent.com/22248748/103560122-3e6ed580-4e85-11eb-9e29-9e31dee972c2.png)

Also, the "offline" indicator, when it appears—i.e., when `PresenceStatusIndicator` gets `false` for its `hideIfOffline` prop—doesn't stand out quite enough when the colors are similar. (I do see that `UserAvatarWithPresence` gives `true` for `hideIfOffline`. But `PresenceStatusIndicator` should do the right thing for all inputs it claims to accept.)

![image](https://user-images.githubusercontent.com/22248748/103561300-40399880-4e87-11eb-8f93-c3da30c20198.png)